web/templates/draft.hbs

```diff
@@ -2,12 +2,14 @@
     <div class="draft-info-box" tabindex="0">
         {{#if is_stream}}
         <div class="message_header message_header_stream">
-            <div class="message-header-contents">
-                <div class="message_label_clickable stream_label {{dark_background}}"
-                  style="background: {{stream_color}}; border-left-color: {{stream_color}};">
+            <div class="message-header-contents" style="background: {{recipient_bar_color}};">
+                <div class="message_label_clickable stream_label">
+                    <span id="stream_sidebar_privacy_swatch_{{stream_id}}" class="stream-privacy filter-icon"  style="color: {{stream_privacy_icon_color}}">
```

Comment:
`{{stream_id}}` isn’t set here, and even if it were, this wouldn’t make sense as an HTML `id` because it’s not unique.

Reply:
Related:

- #24894

Reply:
Thanks!

Reply:
You still need to fix this here.

Reply:
You fixed the other one in `recipient_row.hbs`, but not this one in `draft.hbs`.

Reply:
Should be removed now for sure.